content/contributing/first-time/imposter-syndrome/contents+pt.lr

```diff
@@ -26,7 +26,7 @@ summary: Você pensa que não está pronto para ser um contribuidor open source?
 ---
 gutter:
 
-Connecting with Confident Authenticity
-----------------------------------------------
+Conectando com Autêntica Confiança
+---------------------------------------------
 
-Aprenda mais sobre a sindrome do impostor: assista `"Bake the Cookies, Wear the Dress" <https://www.youtube.com/watch?v=6Uj746j9Heo>`__ by Adrienne Lowe
+Aprenda mais sobre a sindrome do impostor: assista `"Asse os Biscoitos, Vista o Vestido" <https://www.youtube.com/watch?v=6Uj746j9Heo>`__ by Adrienne Lowe
```

Comment:
I'm curious: is it standard practice to translate talk titles from their native language? 

This doesn't make sense to me, as the talk title in English is searchable, where as the translation isn't.